functionaltests/src/test/java/specs/paymentmethods/creditcard/BaseCreditCardChargeFixture.java

```diff
@@ -0,0 +1,34 @@
+package specs.paymentmethods.creditcard;
+
+import io.sphere.sdk.payments.Payment;
+import org.apache.http.HttpResponse;
+import org.concordion.api.MultiValueResult;
+import specs.BaseFixture;
+import specs.paymentmethods.BaseNotifiablePaymentFixture;
+
+import javax.money.format.MonetaryFormats;
+import java.util.Locale;
+
+abstract public class BaseCreditCardChargeFixture extends BaseNotifiablePaymentFixture {
+
+    public MultiValueResult handlePayment(final String paymentName,
+                                          final String requestType) throws Exception {
+        final HttpResponse response = requestToHandlePaymentByLegibleName(paymentName);
+        final Payment payment = fetchPaymentByLegibleName(paymentName);
+        final String transactionId = getIdOfLastTransaction(payment);
+        final String amountAuthorized = (payment.getAmountAuthorized() != null) ?
+                MonetaryFormats.getAmountFormat(Locale.GERMANY).format(payment.getAmountAuthorized()) :
+                BaseFixture.EMPTY_STRING;
+        final String amountPaid = (payment.getAmountPaid() != null) ?
+                MonetaryFormats.getAmountFormat(Locale.GERMANY).format(payment.getAmountPaid()) :
+                BaseFixture.EMPTY_STRING;
+
+        return MultiValueResult.multiValueResult()
+                .with("statusCode", Integer.toString(response.getStatusLine().getStatusCode()))
+                .with("interactionCount", getInteractionRequestCount(payment, transactionId, requestType))
+                .with("transactionState", getTransactionState(payment, transactionId))
+                .with("amountAuthorized", amountAuthorized)
+                .with("amountPaid", amountPaid)
+                .with("version", payment.getVersion().toString());
```

Comment:
don't see a big benefit in this case: these string names are mapped in the HTML fixtures as pure strings, where constants are not possible, see [this](https://github.com/commercetools/commercetools-payone-integration/blob/75558e9d7be867a3f15b477611d9b2e9ef0b7823/functionaltests/src/test/resources/specs/paymentmethods/creditcard/ChargeImmediatelyWith3ds.html#L159-L165)

These are just tests, so keeping it this way is more-or-less fine, as you should associate/verify them in HTML only